requirements/deploy.txt

```diff
@@ -157,7 +157,7 @@ django-storages[boto3]==1.13.1
     # via -r requirements/pip.txt
 django-structlog==2.2.0
     # via -r requirements/pip.txt
-django-taggit==3.0.0
+django-taggit==3.1.0
```

Comment:
Nothing to worry about, https://github.com/jazzband/django-taggit/blob/master/CHANGELOG.rst#310-2022-11-08